doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -538,7 +538,12 @@ Modem libraries
 Multiprotocol Service Layer libraries
 -------------------------------------
 
-|no_changes_yet_note|
+  * Changed implementation of interrupt service rutines wrappers: :c:func:`mpsl_timer0_isr_wrapper`, :c:func:`mpsl_rtc0_isr_wrapper`, :c:func:`mpsl_radio_isr_wrapper`, so that they don't trigger kernel scheduler or don't use any kernel APIs.
```

Comment:
```suggestion
  * Updated the implementation of the following interrupt service routines wrappers:

    * :c:func:`mpsl_timer0_isr_wrapper`
    * :c:func:`mpsl_rtc0_isr_wrapper`
    * :c:func:`mpsl_radio_isr_wrapper`
    
    Now, they do not trigger the kernel scheduler or use any kernel APIs.
```

Reply:
@peknis thank for quick review. The change was applied.